### PR TITLE
feat: add support for @angular/core@18.1.2

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**
[AB#2951511](https://dev.azure.com/blackbaud/Products/_workitems/edit/2951511)

### CODE REVIEW

File: libs/stache/schematics/ng-add/ng-add.schematic.ts [hunk 1]
```diff
@@ -5,9 +5,6 @@ import {
   addPackageJsonDependency,
 } from '@schematics/angular/utility/dependencies';
 
-import fs from 'fs-extra';
```
Comment (Blackbaud-SteveBrush) [inline libs/stache/schematics/ng-add/ng-add.schematic.ts:8]: These changes were done to remove `fs-extra` as a peer dependency.

File: package.json [hunk 2]
```diff
@@ -81,70 +81,70 @@
...
+    "@nx/angular": "19.5.3",
+    "@nx/eslint": "19.5.3",
+    "@nx/eslint-plugin": "19.5.3",
+    "@nx/workspace": "19.5.3",
     "@ryansonshine/commitizen": "4.2.8",
     "@ryansonshine/cz-conventional-changelog": "3.3.4",
-    "@skyux-sdk/testing": "11.0.0-alpha.0",
-    "@skyux/dev-infra-private": "github:blackbaud/skyux-dev-infra-private-builds#10.0.0-alpha.0",
+    "@skyux-sdk/testing": "11.0.0-alpha.6",
+    "@skyux/dev-infra-private": "github:blackbaud/skyux-dev-infra-private-builds#10.0.0-alpha.6",
```
Comment (johnhwhite) [inline package.json:132]: Why not go to `-alpha.9`?

File: .npmrc [hunk 3]
```diff
@@ -1,2 +1 @@
 engine-strict=true
-legacy-peer-deps=true
```
Comment (johnhwhite) [inline .npmrc:2]: Nice.

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "stache",
-  "version": "11.0.0-alpha.0",
+  "version": "11.0.0-alpha.6",
```
Comment (johnhwhite) [inline package.json:3]: Was this intentional?

File: libs/stache/schematics/ng-add/ng-add.schematic.ts [hunk 5]
```diff
@@ -31,16 +28,8 @@ function installEssentialSkyUxPackages(skyuxVersion: string): Rule {
...
-    const packageJson = fs.readJsonSync(
-      path.resolve(__dirname, '../../package.json'),
-    );
-
-    const skyuxVersion = packageJson.peerDependencies['@skyux/core'];
-
     context.addTask(new NodePackageInstallTask());
 
-    return chain([installEssentialSkyUxPackages(skyuxVersion)]);
+    return chain([installEssentialSkyUxPackages('^11.0.0-alpha.6')]);
```
Comment (johnhwhite) [inline libs/stache/schematics/ng-add/ng-add.schematic.ts:33]: Could we drop `fs-extra` without having to bury this hard-coded version?

**Review by johnhwhite — COMMENTED** [review]
Nice work. Couple comments.

File: libs/stache/schematics/ng-add/ng-add.schematic.ts (hunk 5 above)
Comment (Blackbaud-SteveBrush) [reply libs/stache/schematics/ng-add/ng-add.schematic.ts:33]: This value is tested against the value of `VERSION.full` in the unit test, so the test should fail whenever we update the dependency without also updating the version specified, here. I thought it was an okay compromise to having `fs-extra` as a peer dependency.

File: package.json (hunk 4 above)
Comment (Blackbaud-SteveBrush) [reply package.json:3]: Nope!

File: package.json (hunk 2 above)
Comment (Blackbaud-SteveBrush) [reply package.json]: Done.

**Review by johnhwhite — APPROVED** [review]

**Pull request merged**